release/_bom.sh

```diff
@@ -368,6 +368,72 @@ function generate_pom_release_bom_compile_only {
 	) >> "${pom_file_name}"
 }
 
+function generate_pom_release_bom_test {
+	local pom_file_name="release.${LIFERAY_RELEASE_PRODUCT_NAME}.bom.test-${_ARTIFACT_RC_VERSION}.pom"
+
+	lc_log DEBUG "Generating ${pom_file_name}."
+
+	sed \
+		--expression "s/__ARTIFACT_ID__/release.${LIFERAY_RELEASE_PRODUCT_NAME}.bom.test/" \
+		--expression "s/__ARTIFACT_RC_VERSION__/${_ARTIFACT_RC_VERSION}/" \
+		--expression "s/__GITHUB_REPOSITORY__/liferay-${LIFERAY_RELEASE_PRODUCT_NAME}/" \
+		--expression "s/__PRODUCT_VERSION__/${_PRODUCT_VERSION}/" \
+		--expression "w ${pom_file_name}" \
+		"${_RELEASE_TOOL_DIR}/templates/release.bom.pom.tpl" > /dev/null
```

Comment:
@kiwm, please create a new template instead of reuse this one, because it's associated with the BOM named `release.dxp.bom` and it can introduce misunderstanding. :wink: